topics/variant-analysis/tutorials/variant-calling/tutorial.md

```diff
@@ -0,0 +1,605 @@
+---
+layout: tutorial_hands_on
+title: "Variant Calling Workflow"

```

Comment:
Maybe include in the title that this is command-line and not Galaxy.

Reply:
If you put it in the Data science topic under the "Bash" subtopic it will be obvious and doesnt need to change btw

Reply:
Thanks, I moved it under the "bash-variant-calling" subtopic. Is it clear now?